apps/api/src/migration/1653542688339-seedlocationdata.ts

```diff
@@ -0,0 +1,103 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Seedlocationdata1653542688334 implements MigrationInterface {
+  name = 'seedlocationdata1653542688334';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `DELETE FROM public.ien_applicant_jobs_job_location_ien_job_locations WHERE ien_job_locations_id > 84`,
```

Comment:
is this `ien_applicant_jobs_job_location_ien_job_locations` correct?

Reply:
Yes, It is a job-location mapping table(auto-generated when we allow multiple job location selections).
Here we are updating the new location and the max id will be 84. If id>84, those records are not removed then it won't allow us to delete it.
So removing any extra locations that is attached earlier